README.rst

```diff
@@ -5,9 +5,108 @@ Python language bindings for ev3dev
     :target: https://travis-ci.org/rhempel/ev3dev-lang-python
 .. image:: https://readthedocs.org/projects/python-ev3dev/badge/?version=latest
     :target: http://python-ev3dev.readthedocs.org/en/latest/?badge=latest
-    :alt: Documentation Statu
+    :alt: Documentation Status
 
-This is a python library implementing unified interface for ev3dev_ devices.
+A Python library implementing unified interface for ev3dev_ devices.
 
-.. _ev3dev: http://ev3dev.org
+Resources
+---------
+
+`Getting Started with ev3dev <ev3dev-getting-started_>`_
+    If you got here as the result of looking for "how to program
+    LEGO MINDSTORMS EV3 using Python" then you might not be aware that
+    this is part of a much larger project called ev3dev_. Make sure
+    you read the `Getting Started <ev3dev-getting-started_>`_ page
+    to become familiar with ev3dev_ first!
+
+`Connecting the EV3 to the Internet via USB <ev3dev-usb-internet_>`_
```

Comment:
What I'll do is point them at all of the networking options - USB is the minimal method if they don't have a compatible dongle and can't get BT to work. They must have some method of getting to a shell, and USB is the easiest one (IMHO)